src/pages/filter/ui/all-not-mode-view.tsx

```diff
@@ -0,0 +1,34 @@
+import { ReactElement, useMemo } from 'react'
+import { observer } from 'mobx-react-lite'
+
+interface AllNotModeViewProps {
+  filters: Record<string, number>
+}
+
+export const AllNotModeView = observer(
+  ({ filters }: AllNotModeViewProps): ReactElement => {
+    const filtersWithAllMode = useMemo(() => {
+      return Object.keys(filters).includes('All')
+    }, [filters])
+
+    const filtersWithNotMode = useMemo(() => {
+      return Object.keys(filters).includes('Not')
+    }, [filters])
```

Comment:
Convert this to a dumb component that receives only 'not' or 'all' parameter. Or boolean.